bin/oxctl

```diff
@@ -45,7 +46,7 @@ def main():
     if sock in result:
         resp = msgpack.unpackb(sock.recv())
         if options.raw:
-            print resp
+            print json.dumps(resp)
```

Comment:
I think it makes sense to sort the keys when we are making the output 'pretty'.

But for something we call 'raw', probably speed is better. The idea behind this feature is to be able to write scripts against this data, and you should be parsing the JSON, not trying to do some string matching. That would just be asking for trouble.